### PR TITLE
feat(executor): Reduce poll time 3s to 1s for PNS and Emissary executors

### PR DESCRIPTION
**Pull request by alexec**
"So Alex, you appear to have reverted not only your recent changes to the E2E test but also year-old CodeCov changes. A bit embarrassing eh?"

"Ah. Yes. Maybe. But here's why:

* I think I was wrong about separating the E2E tests. The problem is not that they are generally flaky, and so being able to re-run jobs is useful. That was a red-herring. Instead we should fix the specific flakey tests. 
* We removed CodeCov and replaced it with Sonar. But then we lost Sonar, and have nothing. I don't think we need CodeCov for the core team's PR, but I do think it would be useful for PRs submitted by others. CodeCov is quicker to set-up and more robust than SonarQube. 

I also want to make all jobs run in <= 10m (I think this is an infection point for productivity). 

And I want to know which test failed and why without having to open Github Actions (as the UI for actions has poor usability)."

### CODE REVIEW

**Comment by simster7** [comment]
Holding my breath for the results of this race

**Comment by alexec** [comment]
Docker is the winner, by a nose :(

File: tmp/tmp_test.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package tmp
```
Comment (alexec) [inline tmp/tmp_test.go:1]: TODO - remove this file

File: workflow/controller/operator.go [hunk 2]
```diff
@@ -2190,7 +2190,7 @@ func (woc *wfOperationCtx) executeContainer(ctx context.Context, nodeName string
 		return node, err
 	}
 
-	woc.log.Debugf("Executing node %s with container template: %v\n", nodeName, tmpl)
+	woc.log.Debugf("Executing node %s with container template: %v\n", nodeName, tmpl.Name)
```
Comment (alexec) [inline workflow/controller/operator.go:2193]: this made the debug logs noisy and I have never found it useful in the last 1 year

File: workflow/executor/emissary/emissary.go [hunk 3]
```diff
@@ -129,7 +129,7 @@ func (e emissary) Wait(ctx context.Context, containerNames []string) error {
 			if e.isComplete(containerNames) {
 				return nil
 			}
-			time.Sleep(3 * time.Second)
+			time.Sleep(time.Second)
```
Comment (alexec) [inline workflow/executor/emissary/emissary.go:132]: making these loops righter has helped reduce e2e test time and make emissary and PS as fast as Docker

**Comment by alexec** [comment]
> Holding my breath for the results of this race

Docker was the winner, so now I have added changes to make PNs and Emissary as fast/faster.

**Review by simster7 — APPROVED** [review]
No code changes, so: Rubber stamp

**Pull request merged**